### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by maasencioh**
Hello @mljs/collaborators, in the `CONTRIBUTING.md` file I tried to merge all the guidelines, but those are open to discussion off course.

For example will be interesting decide if we make our own commit guidelines, because it's not always clear which case to use, so won't be a bad idea to look other ones as guides, like the [atom's one](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages).

I also think that [we should have a `v1.0.0` version](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages), because we have a stable API and is used in production.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+This method should return plain JSON that enables to reload the current predictor.
+
+### Predictor.load(json)
+
+This static method should return a new predictor instance that is ready to make predictions. The `json`
+parameter is the object returned by an earlier call of `toJSON`.
+
+## Commit Guidelines
+
+The rules are based in the [AngularJS commit guidelines](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit). This leads to **more readable messages** that are easy to follow when looking through the **project history**.
```
Comment (targos) [inline CONTRIBUTING.md]: based on

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+Help us to maximize the effort we can spend fixing issues and adding new
+features, by not reporting duplicate issues.  Providing the following information will increase the
+chances of your issue being dealt with quickly:
+
+* **Overview of the Issue** - if an error is being thrown a non-minified stack trace helps
+* **Motivation for or Use Case** - explain why this is a bug for you
+* **Module Version(s)** - is it a regression?
+* **NodeJS Version(s) and Operating System** - is this a platform related issue?
+* **Reproduce the Error** - provide a live example (using [Tonic](https://tonicdev.com/) or
+  [Visualizer](http://www.cheminfo.org/)) or an unambiguous set of steps.
```
Comment (targos) [inline CONTRIBUTING.md]: https://my.cheminfo.org would be better

**Comment by targos** [comment]
LGTM with a few comments

**Comment by maasencioh** [comment]
Sorry my bad, thanks!

**Comment by targos** [comment]
> I also think that we should have a v1.0.0 version, because we have a stable API and is used in production.

Yes, we can. I'll publish it right away

**Pull request merged**